### PR TITLE
Update Boilerplate Code for Welcome Bot

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+export REPOSITORY=flytepropeller
```
Comment (wild-endeavor) [inline Makefile:1]:
```suggestion
export REPOSITORY=flytekit
```

**Comment by kumare3** [comment]
ping @samhita-alla

**Comment by samhita-alla** [comment]
> ping @samhita-alla

Modified the export line.

File: boilerplate/update.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  echo "***********************************************************************************"
+  echo "$directory is configured in update.cfg."
+  echo "-----------------------------------------------------------------------------------"
+  echo "syncing files from source."
+  rm -rf "${DIR}/${directory}"
+  mkdir -p $(dirname "${DIR}/${directory}")
+  cp -r "$dir_path" "${DIR}/${directory}"
+  if [ -f "${DIR}/${directory}/update.sh" ]; then
+    echo "executing ${DIR}/${directory}/update.sh"
+    "${DIR}/${directory}/update.sh"
```
Comment (pmahindrakar-oss) [inline boilerplate/update.sh:71]: If you meant to execute this then it should be unqouted unless this is allowed by some means now

**Pull request merged**